apps/app/cosmos.tf

```diff
@@ -34,3 +34,11 @@ resource "azurerm_cosmosdb_sql_role_assignment" "cosmos_access" {
   principal_id        = each.value
   scope               = "${data.azurerm_cosmosdb_account.state_store.id}/dbs/${azurerm_cosmosdb_sql_database.app.name}"
 }
+
+resource "azurerm_cosmosdb_sql_container" "example" {
```

Comment:
Do we want to name it `example`? 

What is this container for?

Reply:
Creating container for whatever reason its needed. As devs/data scientist do no have permission to create one.
Renaming it to be just "container" instead as that sounds more appropriate.

Reply:
Done f051821343519211d00dea34441b069ea912c6db